pkg/apis/externaldns/types.go

```diff
@@ -192,8 +194,11 @@ func (cfg *Config) ParseFlags(args []string) error {
 	app.Flag("kubeconfig", "Retrieve target cluster configuration from a Kubernetes configuration file (default: auto-detect)").Default(defaultConfig.KubeConfig).StringVar(&cfg.KubeConfig)
 	app.Flag("request-timeout", "Request timeout when calling Kubernetes APIs. 0s means no timeout").Default(defaultConfig.RequestTimeout.String()).DurationVar(&cfg.RequestTimeout)
 
+	// Flags related to Istio
+	app.Flag("istio-namespace", "The namespace where Istio is installed (default: istio-system)").Default(defaultConfig.IstioNamespace).StringVar(&cfg.IstioNamespace)
```

Comment:
I think we can change that to something like: `--istio-gateway` with the default being `istio-system/istio-ingressgateway` (this way you can configure name and namespace).

Is it realistic to have multiple Istio gateways in your cluster?

Reply:
Great suggestion, I'll make the change.

Reply:
And yes, it's totally realistic (and highly probable) for an Istio cluster to contain multiple Istio Gateways. However, there will only be a single `ingressgateway`.